build.zig

```diff
@@ -187,13 +187,28 @@ pub fn build(b: *std.build.Builder) void {
         );
 
         const unit_tests = b.addTest("src/unit_tests.zig");
+        unit_tests.addPackage(vsr_package);
         unit_tests.setTarget(target);
         unit_tests.setBuildMode(mode);
         unit_tests.addOptions("vsr_options", options);
         unit_tests.step.dependOn(&tb_client_header_generate.step);
         unit_tests.setFilter(test_filter);
         link_tracer_backend(unit_tests, tracer_backend, target);
 
+        // Turn on test coverage if COV env var is not blank.
```

Comment:
This looks familiar. :D

Reply:
@cb22 could you also pull in my updates to HACKING.md that talk about how to use this?

Reply:
Done!